internal/venice-client-common/src/main/java/com/linkedin/venice/stats/dimensions/VeniceMetricsDimensions.java

```diff
@@ -30,6 +30,12 @@ public enum VeniceMetricsDimensions {
   /** {@link com.linkedin.venice.stats.dimensions.MessageType} */
   VENICE_MESSAGE_TYPE("venice.message.type"),
 
+  /** Fanout type for requests {@link com.linkedin.venice.stats.dimensions.RequestFanoutType} (e.g., original vs retry) */
+  VENICE_REQUEST_FANOUT_TYPE("venice.request.fanout_type"),
+
+  /** {@link com.linkedin.venice.stats.dimensions.RejectionReason} */
+  VENICE_REJECTION_REASON("venice.rejection_reason"),
```

Comment:
Any specific reason for chaning this from `venice.request.rejection_reason`?

Reply:
`venice.request.rejection_reason` was not the correct naming. I accidentally add `request` in the middle previously, so just correct it here.